.tiltignore

```diff
@@ -0,0 +1,64 @@
+# Binaries for programs and plugins
+
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Vim swap files
+*.swp**
+
+# Vscode
+.vscode
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+vendor/
```

Comment:
Good catch – yes, I believe we should rebuild the binary in this case. I want to keep the mocks here though, because they are being changed during test runs. I've been experimenting with using tilt to run tests continuously – but changing files by tests during tests results in an infinite loop of constantly running tests… :)